sensors/nxt_i2c_sensor_core.c

```diff
@@ -356,6 +356,33 @@ static int nxt_i2c_sensor_remove(struct i2c_client *client)
 
 static struct i2c_device_id nxt_i2c_sensor_id_table[];
 
+
+static int nxt_i2c_sensor_detect_lego_temp(struct i2c_client *client)
+{
+	int ret;
+	u8 data[1];
+
+	/*
+	 * Try to set the sensor to 12-bit mode.
+	 */
+	data[0] = 0x60;
+	ret = i2c_smbus_write_i2c_block_data(client, 0x01, 1, data);
```

Comment:
Thanks, I was looking for something like that, but via the source. Can you point me to good i2c API docs?